src/NzbDrone.Core.Test/MediaFiles/TrackImport/Specifications/SameFileSpecificationFixture.cs

```diff
@@ -91,5 +92,24 @@ public void should_be_reject_if_file_size_is_the_same()
 
             Subject.IsSatisfiedBy(_localTrack, null).Accepted.Should().BeFalse();
         }
+
+        [Test]
+        public void should_be_accepted_if_file_cannot_be_fetched()
+        {
+            _localTrack.Book = (Book)Builder<Book>.CreateListOfSize(1)
+                .TheFirst(1)
+                .With(e => e.Id = 1)
+                .With(e => e.BookFiles = new LazyLoaded<List<BookFile>>(
```

Comment:
@Qstick  could you check this out for me; it was the last bit that was stopping this from building (just the solution; I am letting Azure run the tests at the moment). Not entirely sure looking at it if this actually does what I am expecting it to do... (the Lidarr equivalent test wasn't much help as track is subtly different from book)

Reply:
The test does fail when I run it manually so guess I have got it wrong :)

Reply:
Yea I'll take a look tonight

Reply:
@robindadswell See if this helps https://github.com/Readarr/Readarr/commit/ab8c78f9aebc41edf58e4169994cedd583ec94b7

It seems to pass tests (I pushed another PR with just the first batch. 

Right now you also have 4 other tests failing (I think a few are fixed in the other branch as well if you want to pick them over) and 52 frontend issues (some things seems to be mixed up)